README.md

```diff
@@ -1,14 +1,14 @@
-# ownCloud Calendar 
+# Calendar 
 
 [![Build Status](https://travis-ci.org/owncloud/calendar.svg?branch=master)](https://travis-ci.org/owncloud/calendar)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 [![Coverage Status](https://coveralls.io/repos/github/owncloud/calendar/badge.svg?branch=master)](https://coveralls.io/github/owncloud/calendar?branch=master)
 [![Dependency Status](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92/badge.svg?style=flat)](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92)
 
-**A calendar app for [ownCloud](https://owncloud.org).**  
+**A calendar app for open cloud platforms like [ownCloud](https://owncloud.org) and [Nextcloud](http://nextcloud.com).**  
```

Comment:
"for ownCloud and Nextcloud" seems to be fine. No need for the "open cloud platforms like" stuff ;)